apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManager.java

```diff
@@ -41,6 +41,7 @@
     private static ThreadLocal<AbstractTracerContext> CONTEXT = new ThreadLocal<AbstractTracerContext>();
     private static ThreadLocal<RuntimeContext> RUNTIME_CONTEXT = new ThreadLocal<RuntimeContext>();
     private static ContextManagerExtendService EXTEND_SERVICE;
+    private static final int NAME_THRESHOLD = 500;
```

Comment:
This should be moved to Config, rather than placing in here.

Reply:
You forget to delete this?

Reply:
Sorry, I made a stupid mistake